### PR TITLE
BG-1346 Donation Builder: style fixes and design improvement

### PR DESCRIPTION
**Pull request by SovereignAndrey**
## Explanation of the solution
- Small style tweaks to donation form builder requested:
  - Swap order of preview and input form on page (preview now on left-most side)
  - Add more padding to final line of copy
- Change copy/design of the Title over the Preview section (now has Laira Character and more eye-catching font)
  - Needed to note to users it's a live preview, since it is largely being ignored by user (hotjar playbacks). :sweat_smile:  

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
- navigate to donation builder page and inspect the new layout changes

## UI changes for review
donation-builder

### CODE REVIEW

File: src/pages/Widget/Preview.tsx [hunk 1]
```diff
@@ -11,9 +13,12 @@ export default function Preview({ classes = "" }) {
 
   return (
     <section className={`${classes} @container/preview pb-4`}>
-      <h2 className="text-lg @4xl/widget:text-2xl text-center @4xl/widget:text-left mb-3">
-        That's what our Donation Form looks like:
-      </h2>
+      <span className="max-sm:hidden">
```
Comment (ap-justin) [inline src/pages/Widget/Preview.tsx:16]:
```suggestion
      <div className="max-sm:hidden">
```
nit: inline element `span` shoudn't contain block element `p`

File: src/pages/Widget/Preview.tsx [hunk 2]
```diff
@@ -11,9 +13,12 @@ export default function Preview({ classes = "" }) {
...
+        <p className="flex text-navy-d4 text-2xl font-gochi text-nowrap">
+          <Image src={character} className="h-[45px] mr-2 pb-2" />
+          Check out the LIVE preview of your Donation Form!
```
Comment (ap-justin) [inline src/pages/Widget/Preview.tsx:19]: this text doesn't have enough space around 
image

original 
image

**Review by ap-justin — CHANGES_REQUESTED** [review]

File: src/pages/Widget/Preview.tsx (hunk 2 above)
Comment (SovereignAndrey) [reply src/pages/Widget/Preview.tsx:19]: @ap-justin Good catch! I'll fix that.

File: src/pages/Widget/Preview.tsx (hunk 1 above)
Comment (SovereignAndrey) [reply src/pages/Widget/Preview.tsx]: Fixed in 07587398.

File: src/pages/Widget/Preview.tsx (hunk 2 above)
Comment (SovereignAndrey) [reply src/pages/Widget/Preview.tsx:19]: Fixed in 4609260c. Also fixed issues with the new text not wrapping & the snippet alignment with it's title.

File: src/pages/Widget/Preview.tsx [hunk 3]
```diff
@@ -10,10 +12,13 @@ export default function Preview({ classes = "" }) {
   const endowName = endowment.name || "this nonprofit";
 
   return (
-    <section className={`${classes} @container/preview pb-4`}>
-      <h2 className="text-lg @4xl/widget:text-2xl text-center @4xl/widget:text-left mb-3">
-        That's what our Donation Form looks like:
-      </h2>
+    <section className={`${classes} @container/preview pb-4 mt-10`}>
```
Comment (ap-justin) [inline src/pages/Widget/Preview.tsx:15]:
```suggestion
    <section className={`${classes} @container/preview pb-4`}>
```
classes affecting outside space should be outside e.g.

```jsx
 <div className="w-full">
      <Configurer endowment={endowment} />
      <Snippet classes="mt-10" />
 </div>
 <Preview classes="mt-10 .... existing classes" />
```

**Review by ap-justin — CHANGES_REQUESTED** [review]

**Comment by SovereignAndrey** [comment]
Thanks for the review @ap-justin. Fixed both items in 84669b95.

**Review by ap-justin — APPROVED** [review]

**Pull request merged**